src/MyHealth/History.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { FunctionComponent } from "react"
+import { ScrollView, StyleSheet } from "react-native"
+
+import { useSymptomLogContext } from "./SymptomLogContext"
+import { Typography, Colors, Outlines, Spacing } from "../styles"
+import Calendar from "./Calendar"
+import { toLogDataHistory } from "./logDataHistory"
+import DaySummary from "./DaySummary"
+import { DayLogData } from "./symptoms"
+
+const History: FunctionComponent = () => {
+  const {
+    dailyLogData,
+    selectLogEntry,
```

Comment:
what do you think about having the `selectLogEntry` and `selectedLogEntry` live on the History component itself? I dont think any other component will need access to these and we probably want to minimize the surface area of state things.